com.unity.multiplayer.mlapi/Runtime/Messaging/RpcQueue/RpcQueueContainer.cs

```diff
@@ -703,22 +661,27 @@ private void ClearParameters()
         }
 
         /// <summary>
-        /// Shutdown
         /// Flushes the internal messages
         /// Removes itself from the network update loop
         /// Disposes readers, writers, clears the queue history, and resets any parameters
         /// </summary>
-        public void Shutdown()
+        private void Shutdown()
         {
+            if (NetworkLog.CurrentLogLevel == LogLevel.Developer)
+            {
+                NetworkLog.LogInfo($"[Instance : {s_RpcQueueContainerInstances}] {nameof(RpcQueueContainer)} shutting down.");
```

Comment:
Let's surround this with `#if UNITY_EDITOR || DEVELOPMENT_BUILD`

Reply:
Also wrapped the property (and triple checked it was not used anywhere else)